packages/flutter/lib/src/widgets/drag_target.dart

```diff
@@ -372,6 +388,10 @@ class _DraggableState<T> extends State<Draggable<T>> {
           _activeCount -= 1;
           _disposeRecognizerIfInactive();
         }
+        if(mounted && widget.onDragEnd != null)
```

Comment:
nit: but a space between `if` and `(`, since it's not a method call

Reply:
Consider it done.